services/vision/detectionstosegments/detections_to_3dsegments_test.go

```diff
@@ -41,8 +40,8 @@ func Test3DSegmentsFromDetector(t *testing.T) {
 	cam.ImagesFunc = func(ctx context.Context) ([]camera.NamedImage, resource.ResponseMetadata, error) {
 		return nil, resource.ResponseMetadata{}, errors.New("no images")
 	}
-	cam.ProjectorFunc = func(ctx context.Context) (transform.Projector, error) {
-		return &transform.ParallelProjection{}, nil
```

Comment:
@bhaney will need your help here, I dumbly changed it to camera.Properties but I suspect I need some actual properties to get two objects back.

Reply:
I can check out your PR and edit the test